cli/flow-typed/jest.js

```diff
@@ -0,0 +1,55 @@
+type JestMockFn = {
```

Comment:
Are we sure we need to add type definitions like this twice in this repo? I would opt for a copy command in a postinstall script. Possibly using some node module to ensure it also works on windows.